net/sysrepo/Makefile

```diff
@@ -21,25 +21,27 @@ PKG_LICENSE_FILES:=LICENSE
 
 CMAKE_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
-PKG_BUILD_DEPENDS:=SYSREPO_BINDINGS:swig/host
+PKG_BUILD_DEPENDS:=swig/host
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
-ifeq ($(CONFIG_SYSREPO_PYTHON),y)
-$(call include_mk, python3-package.mk)
-endif
-
-define Package/libsysrepo/config
-	source "$(SOURCE)/Config_libsysrepo.in"
-endef
+include ../../lang/python/python3-package.mk
 
 define Package/libsysrepo
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=YANG-based data store library
   URL:=https://www.sysrepo.org/
-  DEPENDS:=+libyang +libprotobuf-c +libev +libredblack +librt +libpthread +SYSREPO_BINDINGS:libstdcpp +SYSREPO_PYTHON:python3-base +SYSREPO_LUA:lua
-  MENU:=1
+  DEPENDS:=+libyang +libprotobuf-c +libev +libredblack +librt +libpthread
+endef
+
+define Package/python3-sysrepo
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=YANG-based data store library - Python 3 bindings
+  URL:=https://www.sysrepo.org/
+  DEPENDS:=+libsysrepo +libstdcpp +python3-base
```

Comment:
libstdc;pp should be implicitly included with libsysrepo.

Reply:
If you mean that libstdcpp should already be a dependency of one of libsysrepo's dependencies, this doesn't appear to be the case. (I've installed this libsysrepo by itself and it doesn't pull in libstdcpp.)

If you mean the `+libstdcpp` should be moved from the python3-sysrepo `DEPENDS` to libsysrepo, I don't think libsysrepo has a real dependency on libstdcpp. The compiled module for the Python bindings (`_sysrepo.so`) depends on the C++ bindings (`libSysrepo-cpp.so`), and it's that C++ binding that depends on libstdcpp.

Reply:
Derp. I confused libyang with libyamp-cpp.

From looking at the code on github, it seems C++ is used in swig. I tested uClibc++. Unfortunately, it's using shared_ptr, which is not compatible.